src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -584,7 +658,7 @@ export class VSCodeNotebookController implements Disposable {
             isLocalConnection(this.kernelConnection)
         ) {
             // Startup could fail due to missing dependencies or the like.
-            void newKernel.start({ disableUI: true });
+            this.connectToKernel(document, new DisplayOptions(true)).catch(noop);
```

Comment:
THis was another fix, here we start `newKernel.start` our selves, but `connectToKernel` function will do something else and those two would step on their toes.